scripts/rollback-manual.sh

```
@@ -0,0 +1,53 @@
+LATEST_BACKUP=$(ssh $SERVER "cd $REMOTE_DIR && ls -t backups/*.tar.gz 2>/dev/null | head -1")
+
+if [ -z "$LATEST_BACKUP" ]; then
+    print_error "No backup found! Cannot rollback."
+    exit 1
+fi
+
+print_status "Rolling back from backup: $LATEST_BACKUP"
+
+# Stop current services
+print_status "Stopping current services..."
+ssh $SERVER "cd $REMOTE_DIR && docker-compose down --remove-orphans || true"
```

Shellcheck (0.10.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)